### ISSUE TITLE
Cop idea: `Hash#fetch` defaults

### ISSUE THREAD

**Issue by ydakuka** [rubocop/rubocop#12345]
## Description

Introduce default values for hash keys via Hash#fetch as opposed to using custom logic.

https://rubystyle.guide/#hash-fetch-defaults

## Describe the solution you'd like

```ruby
batman = { name: 'Bruce Wayne', is_evil: false }

# bad - if we just use || operator with falsey value we won't get the expected result
batman[:is_evil] || true # => true

# good - fetch works correctly with falsey values
batman.fetch(:is_evil, true) # => false
```
